server/controllers/admin/grades.js

```diff
@@ -0,0 +1,152 @@
+/**
+* Grade  Controller
+*
+* This controller exposes an API to the client for reading and writing Grade 
+
+*/
+var db = require('../../lib/db');
+var uuid = require('../../lib/guid');
```

Comment:
Please use `node-uuid` instead.  `uuid` is deprecated.  There should be a message in your console about it.